api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/util/Wrappers.java

```diff
@@ -75,4 +79,20 @@ public static void setupTenantRoute(Router router, HttpMethod method, String pat
                 },
                 ctx.response()::end);
     }
+
+    public static Transformer<String, String> toJsonArray(RoutingContext ctx) {
```

Comment:
@burmanm Can you provide a link/reference to that example? I don't see it mentioned anywhere in the PR comments.